app/experimenter/experiments/models/nimbus.py

```diff
@@ -484,26 +484,48 @@ def clone(self, name, user):
         cloned.reference_branch = None
         cloned.published_dto = None
         cloned.results_data = None
+        if rollout_branch_slug:
+            cloned.proposed_duration = NimbusExperiment.DEFAULT_PROPOSED_DURATION
+            cloned.proposed_enrollment = NimbusExperiment.DEFAULT_PROPOSED_ENROLLMENT
+            cloned.population_percent = 0
+            cloned.total_enrolled_clients = 0
         cloned.save()
 
-        for branch in self.branches.all():
+        if rollout_branch_slug:
```

Comment:
We do `if rollout_branch_slug:` just above this on L487-491, should we merge these two blocks?

Reply:
Yeah probably could fold those above lines into this block.